ci/fluentd.rb

```diff
@@ -9,7 +9,7 @@
     end
 
     task :before_script => ['ci:common:before_script'] do
-      sh %(fluentd -c $TRAVIS_BUILD_DIR/ci/resources/fluentd/td-agent.conf &)
+      sh %(fluentd -c $TRAVIS_BUILD_DIR/ci/resources/fluentd/td-agent.conf > /dev/null 2>&1 &)
```

Comment:
Is this command really spammy ? 

Won't we be completely blind if it starts failing without us realizing ?